src/engine-actions/create-wallet.js

```diff
@@ -3,25 +3,57 @@ const {
   initWallet
 } = require('../lnd-actions')
 
+/**
+ * Wallet recovery window number in blocks
+ * @constant
+ * @type {number}
+ * @default
+ */
+const RECOVERY_WINDOW_DEFAULT = 5000
+
+/**
+ * @param {Array<string>} seed
+ * @returns {boolean}
+ */
+async function isValidSeed (seed) {
+  if (!Array.isArray(seed)) return false
+  if (seed.length !== 24) return false
+  return true
+}
+
 /**
  * Creates a wallet
  *
  * @param {string} password - wallet password, used to unlock lnd wallet
+ * @param {Object} opts
+ * @param {Array<string>} opts.seed - 24 word cipher seed mnemonic
+ * @param {Buffer} opts.backup - binary string of an lnd backup file
  * @returns {Array<string>} 24 word cipher seed mnemonic
  */
-async function createWallet (password) {
-  const { cipherSeedMnemonic } = await genSeed({ client: this.walletUnlocker })
-
-  if (typeof password !== 'string') {
-    throw new Error('Provided password must be a string value')
+async function createWallet (password, { seed, backup, recoveryWindow = undefined } = {}) {
```

Comment:
Did you consider making this two different engine actions? Although they are the same (`initWallet`) on LND, they feel pretty different to me from a user (and implementation) perspective.

Reply:
@treygriffith I did have it as a second action. TBH it would be better that way. will make the change